cmd/main.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
No pongáis nunca una función main. Y menos "por probar"

Reply:
Eliminado main.go